### ISSUE TITLE
id.ts won't work for packages that don't run on linux/x86-64

### ISSUE THREAD

**Issue by jhheider** [pkgxdev/brewkit#278]
https://github.com/pkgxdev/pantry/actions/runs/7588437583/job/20670951786

```
Run pkgxdev/brewkit/id@v1
Run if ! pkgx --sync; then

Run echo "::group::fetch deno deps"
  echo "::group::fetch deno deps"
  pkgx deno cache ${GITHUB_ACTION_PATH}/id.ts --config=${GITHUB_ACTION_PATH}/../deno.jsonc
  echo "::endgroup::"
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
fetch deno deps
Run ${GITHUB_ACTION_PATH}/id.ts macvim.org=7.4.017
  ${GITHUB_ACTION_PATH}/id.ts macvim.org=7.4.017
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    GH_TOKEN: ***
warn: project not available on this platform
Error: Process completed with exit code 2.
```

**Comment by jhheider** [comment]
~hm, works locally for `wayland.freedesktop.org/protocols`. investigating.~

```sh
% GITHUB_OUTPUT=foo GITHUB_ENV=foo ./id/id.ts wayland.freedesktop.org/protocols=1.3
3.0
error: Uncaught (in promise) DownloadError: http: 404: https://dist.pkgx.dev/wayland.freedesktop.org/darwin/aarch64/versions.txt
    throw new DownloadError(rsp.status, {src: url})
          ^
    at Object.get (https://deno.land/x/libpkgx@v0.16.0/src/hooks/useInventory.ts:32:11)
    at eventLoopTick (ext:core/01_core.js:181:11)
    at async Object.select (https://deno.land/x/libpkgx@v0.16.0/src/hooks/useInventory.ts:17:20)
    at async Promise.all (index 0)
    at async Object.resolve (https://deno.land/x/libpkgx@v0.16.0/src/plumbing/resolve.ts:68:3)
    at async construct_config (file:///Users/jacob/pkgx/brewkit/lib/config.ts:72:17)
    at async config (file:///Users/jacob/pkgx/brewkit/lib/config.ts:59:12)
    at async file:///Users/jacob/pkgx/brewkit/id/id.ts:5:16
```
likely this can just be caught so we can config even if we don't run on linux/x86-64

**Comment by jhheider** [comment]
additional examples:
https://github.com/pkgxdev/pantry/actions/runs/7588460858
https://github.com/pkgxdev/pantry/actions/runs/7588460869
https://github.com/pkgxdev/pantry/actions/runs/7588460904
https://github.com/pkgxdev/pantry/actions/runs/7588491867

**Comment by mxcl** [comment]
My bad, I don't believe this has any reason to be platform specific so let me fix it properly.

**Comment by mxcl** [comment]
Fixed in #280
